docker-compose.yaml

```diff
@@ -10,15 +10,34 @@ services:
     ports:
       - 5432:5432
   calendso:
-    build: .
+    build:
+      context: .
+      dockerfile: Dockerfile
+      args:
+        - BASE_URL=${BASE_URL}
+        - NEXT_PUBLIC_APP_URL=${NEXT_PUBLIC_APP_URL}
+    image: calendso/docker
     restart: always
     ports:
       - 3000:3000
+    env_file: .env
+    environment:
+      - DATABASE_URL=postgresql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@${DATABASE_HOST}/${POSTGRES_DB}
+    depends_on:
+      - db
+  studio:
```

Comment:
My current understanding is that prisma studio is only needed to set up the first user. The calendso webapp works just fine without it. 
For development, this is probably nice to have, but for deployment, this is superfluous (and potentially dangerous if the port is open to the public).

Reply:
This is definitely a step forward with the same motivations, and why I wanted to break it out to its own thing.  Do you think it's enough to add a comment before the `studio` service definition about that? or should we comment out the service by default?

(side personal opinion, no one should be using docker-compose in production)

Reply:
Sorry for the late comment. My notification settings were messed up.

Long-term, I think I'd comment out the service. It would be cool to just _automatically_ create an initial user account, instead. But for now, I think this is okay.

(Why not use docker-compose in production? I feel like it's not really that different than me just starting the containers manually. For my own use-case, I don't really need any scalability.)

Reply:
I'm err on the side of using 'production' as a term to specify an enterprise HA environment. 

Side note, Docker (the branded cli from Docker) is being deprecated by most if not all of the enterprise cloud providers in favor of the open source container runtimes, and docker-compose is part of the Docker brand-specific tooling. (but it's still the best tool for local container development imo)

Reply:
My preferred pattern is actually to move optional services and configs to a docker-compose.override.yml file and use docker-compose.yml as the "production" options. Maybe that's a way to move too? onwards and upwards!